README.md

```diff
@@ -11,14 +12,15 @@
 From the root of the cloned or forked repository, run this command:
 
 ```
-npm install && ./and-cli.js install
+npm install && npm run build && ./dist/and-cli.js install
 ```
 
 What this does:
 
+1. Installs dependencies and runs a TypeScript build of the project
 1. Installs a global version of the npm package
-2. Creates an alias `and-cli` for the installed version of the npm package in your project, which defaults to globally installed package if not installed for project
-3. Creates an alias `and-cli-dev` for the running the cli while developing for it, via the directory in which you cloned or forked the repository
+1. Creates an alias `and-cli` for the installed version of the npm package in your project, which defaults to globally installed package if not installed for project
+1. Creates an alias `and-cli-dev` for the running the cli while developing for it, via the directory in which you cloned or forked the repository
```

Comment:
Feels like the "the" before "running" shouldn't be there. Maybe I'm just reading it wrong.

Reply:
Good catch 🤦